metricbeat/module/redis/info/_meta/fields.yml

```diff
@@ -185,31 +289,82 @@
       fields:
         - name: role
           type: keyword
-          description:
+          description: >
+            Role of the instance (can be "master", or "slave").
 
         - name: connected_slaves
           type: long
-          description:
+          description: >
+            Number of connected slaves
 
         - name: master_offset
           type: long
-          description:
-
+          deprecated: true
```

Comment:
After merging #7779 today, `deprecated` needs to be set to the version the field is going to be deprecated on, in this case set it to 6.5.